packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/index.stories.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { storyWithProps } from '../../storybook-email';
+
+export default {
+  title: 'Emails/passwordResetAccountRecovery',
+} as Meta;
+
+const createStory = storyWithProps(
+  'passwordResetAccountRecovery',
+  'Sent when recovery key is used',
+  {
+    location: 'Madrid, Spain (estimated)',
```

Comment:
I know you know this too but you can use `...MOCK_LOCATION` now + in your other `.stories` file 😎